packages/rocketchat-livechat/server/hooks/sendToCRM.js

```diff
@@ -5,15 +5,24 @@ function sendToCRM(type, room, includeMessages = true) {
 
 	postData.messages = [];
 
-	if (includeMessages) {
-		RocketChat.models.Messages.findVisibleByRoomId(room._id, { sort: { ts: 1 } }).forEach((message) => {
+	let messages;
+	if (typeof includeMessages === 'boolean' && includeMessages) {
+		messages = RocketChat.models.Messages.findVisibleByRoomId(room._id, { sort: { ts: 1 } });
+	} else if (includeMessages instanceof Array) {
+		messages = includeMessages;
+	}
+
+	if (messages) {
+		messages.forEach((message) => {
 			if (message.t) {
 				return;
 			}
 			const msg = {
```

Comment:
@sampaiodiego here I added "_id" and "editedAt" fields. This is the payload sent on webhook request.